content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+pvc-e8aed33b-dc99-41bc-b301-2e2d3ca81b8d   30Gi       RWO            Delete           Bound    ashish-hpe-mongo-deploy/ashish-mongo-persistent-storage-mongo-0                                                       gl-sbc-hpe                       47h
+pvc-ee3146e2-ad77-4418-93b8-a94cd30e63ad   10Gi       RWO            Delete           Bound    ashish-mongo-deploy/nimblepod-10                                                                                      gl-sbc-hpe                       66d
+pvc-eee0e074-3284-40f5-914e-88842899efc2   5Gi        RWO            Delete           Bound    ashish-mongo-deploy/ashish-test-pvc                                                                                   gl-sbc-hpe                       69d
+pvc-f42472d1-f8f2-4ac4-8955-c37664d7cf8b   30Gi       RWO            Delete           Bound    ashish-hpe-mongo-deploy/ashish-mongo-persistent-storage-mongo-1                                                       gl-sbc-hpe                       47h
+
+
+$ kubectl describe pv pvc-bc7faca6-6cd6-4796-acc1-864716c98f86
+Name:            pvc-bc7faca6-6cd6-4796-acc1-864716c98f86
+Labels:          <none>
+Annotations:     pv.kubernetes.io/provisioned-by: csi.hpe.com
+Finalizers:      [kubernetes.io/pv-protection external-attacher/csi-hpe-com]
+StorageClass:    gl-sbc-glhcnimblestor
+Status:          Bound
+Claim:           mongo/mongo-persistent-storage-mongo-0
+Reclaim Policy:  Delete
+Access Modes:    RWO
+VolumeMode:      Filesystem
+Capacity:        50Gi
+Node Affinity:   <none>
+Message:
+Source:
+    Type:              CSI (a Container Storage Interface (CSI) volume source)
+    Driver:            csi.hpe.com
+    FSType:            xfs
+    VolumeHandle:      067ac428a6431c485b00000000000000000000044d
+    ReadOnly:          false
+    VolumeAttributes:      accessProtocol=iscsi
+                           allowOverrides=nfsResources,nfsNamespace
+                           cloneOf=
+                           createSnapshot=false
+                           dedupeEnabled=true
+                           description=hpe
+                           destroyOnDelete=true
+                           encrypted=false
+                           folder=caas-pvs
+                           fsType=xfs
+                           hostEncryption=false
+                           limitIops=-1
+                           limitMbps=-1
+                           performancePolicy=default
+                           pool=default
+                           storage.kubernetes.io/csiProvisionerIdentity=1663147353961-8081-csi.hpe.com
+                           syncOnDetach=false
+                           targetScope=volume
+                           thick=false
+                           volumeAccessMode=mount
+Events:                <none> 
+```
+
+### Step-8: Configure MongoDB Primary and Secondary Replica
```

Comment:
A little explanation beneath this header would also be appropriate. Do "Primary", "Secondary" and "Replica" all need initial capital letters? Are you referring to specific names of things, things that are found on a screenshot that are capitalized that way, or are these just basic nouns?

Reply:
done